docs/pages/versions/unversioned/sdk/dataMigrationHelper.md

```diff
@@ -0,0 +1,79 @@
+---
+title: DataMigrationHelper
+---
+
+**`DataMigrationHelper`** helps you to solve a problem with missing files after SDK upgrade on Android.
+The problem only occurs when the ejected/standalone application is upgraded from SDK 32 or below to SDK 33 or above.
+
+## API
+
+```js
+import { DataMigrationHelper } from 'expo';
+```
+
+### `DataMigrationHelper.getLegacyDocumentDirectoryAndroid(): string | null`
+
+Returns the path to the legacy document directory which can be used by `expo-file-system`.
+
+> Returns null on all platforms except Android
+
+### `DataMigrationHelper.migrateFilesFromLegacyDirectoryAsync(conflictResolver?: ConflictResolver): Promise<void>`
+
+```js
+
+export default class App extends React.Component {
+  state = {
+    movedFiles: false,
+  };
+
+  render() {
+    const { movedFiles } = this.state;
+
+    if (!movedFiles) {
+      return (
+        <AppLoading
+          startAsync={this._moveFiles}
+          onFinish={() => this.setState({ movedFiles: true })}
+          onError={console.warn}
+        />
+      );
+    }
+
+    return (
+      ...
+    );
+  }
+
+  async _moveFiles() {
+    await DataMigrationHelper.migrateFilesFromLegacyDirectoryAsync(); 
+  }
+  ...
+}
+
+```
+
+This method moves all files from the legacy directory to the current document directory. 
+`conflictResolver` argument is optional and if it is possible to leave it empty we encourage to do so because the method runs faster without it.
```

Comment:
```suggestion
The `conflictResolver` argument is optional, and if it is possible to leave it empty, we encourage doing so because the method runs faster without it.
```